build.sc

```diff
@@ -11,15 +11,18 @@ import mill._
 import mill.scalalib._
 import mill.scalalib.publish.{Developer, License, PomSettings, VersionControl}
 
-object `kubernetes-client` extends Cross[KubernetesClientModule]("2.13.4", "2.12.12")
+object `kubernetes-client` extends Cross[KubernetesClientModule]("2.13.5", "2.12.12")
 class KubernetesClientModule(val crossScalaVersion: String)
     extends CrossScalaModule
     with TpolecatModule
     with StyleModule
     with GitVersionedPublishModule
     with SwaggerModelGenerator {
 
-  override def scalacOptions = super.scalacOptions().filter(_ != "-Wunused:imports")
+  override def scalacOptions = super.scalacOptions()
+    .filter(_ != "-Wunused:imports")
+    .filter(_ != "-Xfatal-warnings")
```

Comment:
I could not compile it with disabling "warning as errors" flag. If I switch the project to 2.13.5, then there we get warning that some Circe Encoced/Decoder codecs are unused, however they are really used. I am not sure who is checking for unused implicits, is it scalac or scalafix.

Reply:
This is an example of unused implicits: https://github.com/joan38/kubernetes-client/pull/90/checks?check_run_id=2904560216#step:5:798